packages/cozy-client/src/CozyClient.js

```diff
@@ -576,8 +576,7 @@ client.query(Q('io.cozy.bills'))`)
     // Don't trigger the INIT_QUERY for fetchMore() calls
     if (
       existingQuery.fetchStatus !== 'loaded' ||
-      !queryDefinition.skip ||
-      !queryDefinition.bookmark
+      (!queryDefinition.skip && !queryDefinition.bookmark)
```

Comment:
can we have a test for this? ;)

Reply:
Done with https://github.com/cozy/cozy-client/pull/670/commits/ffd4843a75e79bdb386d3d48ac942e6b31fdd007 ;)